src/imgui/ImGuiUtils.hh

```diff
@@ -48,6 +50,20 @@ inline auto CalcTextSize(std::string_view str)
 	return ImGui::CalcTextSize(str.data(), str.data() + str.size());
 }
 
+inline std::string leftClip(std::string_view s, float maxWidth)
+{
+	auto fullWidth = ImGui::CalcTextSize(s).x;
+	if (fullWidth <= maxWidth) return std::string(s);
+
+	maxWidth -= ImGui::CalcTextSize("..."sv).x;
```

Comment:
This `sv` suffix is the only user of `using namespace std::literals`.

If we want to avoid that global `using namespace` in this `.hh` file, then we could move it from the global scope to inside this function (when it's local to this function it can no longer affect other unknown code).
Or if it's just this single use then we could also write `std::string_view("...")`.